src/lib/constants.js

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable */
+export const GAS_LIMIT = 6000000;
+export const DEFAULT_ADMIN_ROLE_PUBLIC_ADDRESS =
```

Comment:
Minor but these are not addresses, they are hashes:
`keccack256("NAME_OF_THE_ROLE")`

Reply:
updated